qemu/tests/cfg/blockdev_inc_backup_with_ignore.cfg

```diff
@@ -0,0 +1,44 @@
+- blockdev_inc_backup_with_ignore:
+    only Linux
+    start_vm = no
+    kill_vm = yes
+    qemu_force_use_drive_expression = no
+    type = blockdev_inc_backup_with_ignore
+    virt_test_type = qemu
+    images += " data1"
+    source_images = data1
+    image_backup_chain_data1 = "full inc"
+    remove_image_data1 = yes
+    force_create_image_data1 = yes
+    storage_pools = default
+    storage_pool = default
+    storage_type_default = directory
+    full_backup_options = '{"sync": "full"}'
+    inc_sync_mode = incremental
+    on_target_error = ignore
+    tempfile_size = 500M
+    inc_node = "drive_inc"
+
+    image_size_data1 = 2G
+    image_size_full = ${image_size_data1}
+    image_format_data1 = qcow2
+    image_format_full = qcow2
+    image_name_data1 = images/data1
+    image_name_full = full
+    rebase_mode = unsafe
+
+    # lvm setting
+    mount_path = /var/tmp/mnt
+    vg_name = vg_test
+    lv_name = lv_test
+    lv_size = 150M
+    block_raw_name = lvm_block
+    emulated_image = images/${block_raw_name}
+    emulated_image_size = ${image_size_data1}
+    lv_extend_cmd = "lvextend -L ${emulated_image_size} /dev/${vg_name}/${lv_name} && resize2fs /dev/${vg_name}/${lv_name}"
```

Comment:
Add the following:
force_remove_image = yes
remove_emulated_image = yes
to remove vg/pv and loop device

Reply:
Have updated, thanks zhenchao,